test/lock-bot.test.ts

```diff
@@ -265,6 +279,26 @@ const runAllTests = () => {
     });
   });
 
+  test("can lock resource with message (single word, i.e no space)", async () => {
+    expect(await execute("/lock dev blahblah")).toEqual({
+      message: "<@Connor> has locked `dev` 🔒",
+      destination: "channel",
+      metaData: {
+        Message: "blahblah",
+      },
+    });
+  });
+
+  test("can lock resource with message (multi word, i.e no space)", async () => {
+    expect(await execute("/lock dev blahblah test")).toEqual({
+      message: "<@Connor> has locked `dev` 🔒",
+      destination: "channel",
+      metaData: {
+        Message: "blahblah test",
+      },
+    });
+  });
+
```

Comment:
If you try to get the `description` after storing it will not work because there's no code to retrieve it.
It only shows the description when you lock it but not when you type `/locks` or try and get the description from the API.

If you try this test it will fail
```typescript
  test("Create lock with description, get lock", async () => {
    await server
      .post("/dev/api/teams/T012345WXYZ/channels/C012345ABCD/locks")
      .set("Authorization", `Basic ${credentials1}`)
      .send('{ "name": "dev", "owner": "U012345MNOP", "message": "Hey man" }');

    const res = await server
      .get("/dev/api/teams/T012345WXYZ/channels/C012345ABCD/locks/dev")
      .set("Authorization", `Basic ${credentials1}`);

    expect(res.status).toBe(200);
    expect(res.text).toBe(
      JSON.stringify({
        name: "dev",
        owner: "U012345MNOP",
        message: "Hey man",
      })
    );
  });
```